2013/07/08 14:50:22 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:50:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 D:\wamp\www\yii\demos\blog\protected\models\User.php(20): CActiveRecord::model('User')
#8 D:\wamp\www\yii\demos\blog\protected\components\UserIdentity.php(18): User::model()
#9 D:\wamp\www\yii\demos\blog\protected\models\LoginForm.php(50): UserIdentity->authenticate()
#10 D:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#11 D:\wamp\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#12 D:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#13 D:\wamp\www\yii\demos\blog\protected\controllers\SiteController.php(82): CModel->validate()
#14 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 D:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/site/login
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:50:24 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:50:24 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:50:29 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:50:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 D:\wamp\www\yii\demos\blog\protected\models\User.php(20): CActiveRecord::model('User')
#8 D:\wamp\www\yii\demos\blog\protected\components\UserIdentity.php(18): User::model()
#9 D:\wamp\www\yii\demos\blog\protected\models\LoginForm.php(50): UserIdentity->authenticate()
#10 D:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#11 D:\wamp\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#12 D:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#13 D:\wamp\www\yii\framework\web\widgets\CActiveForm.php(835): CModel->validate(NULL)
#14 D:\wamp\www\yii\demos\blog\protected\controllers\SiteController.php(73): CActiveForm::validate(Object(LoginForm))
#15 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#16 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#17 D:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#18 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#20 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#21 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#23 {main}
REQUEST_URI=/yii/demos/blog/index.php/site/login
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:50:31 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:50:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 D:\wamp\www\yii\demos\blog\protected\models\User.php(20): CActiveRecord::model('User')
#8 D:\wamp\www\yii\demos\blog\protected\components\UserIdentity.php(18): User::model()
#9 D:\wamp\www\yii\demos\blog\protected\models\LoginForm.php(50): UserIdentity->authenticate()
#10 D:\wamp\www\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#11 D:\wamp\www\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#12 D:\wamp\www\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#13 D:\wamp\www\yii\demos\blog\protected\controllers\SiteController.php(82): CModel->validate()
#14 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#15 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 D:\wamp\www\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('login')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/site/login
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:51:41 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:51:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:51:43 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:51:43 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/post/index
---
2013/07/08 14:51:44 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:51:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/post/index
---
2013/07/08 14:51:45 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:51:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/post/index
---
2013/07/08 14:55:06 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/post/index
---
2013/07/08 14:55:08 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/post/index
---
2013/07/08 14:55:15 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:55:17 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:55:17 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 14:55:17 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2013/07/08 14:55:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\wamp\www\yii\framework\db\CDbConnection.php:387
Stack trace:
#0 D:\wamp\www\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\wamp\www\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\wamp\www\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\wamp\www\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#7 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#8 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#9 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#10 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#17 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#20 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#22 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/08 15:03:25 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'blog.tbl_post' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_post`.
2013/07/08 15:03:25 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{post}}" for active record class "Post" cannot be found in the database.' in D:\wamp\www\yii\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 D:\wamp\www\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Post))
#1 D:\wamp\www\yii\framework\web\CActiveDataProvider.php(74): CActiveRecord::model('Post')
#2 D:\wamp\www\yii\demos\blog\protected\controllers\PostController.php(132): CActiveDataProvider->__construct('Post', Array)
#3 D:\wamp\www\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#4 D:\wamp\www\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\wamp\www\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\wamp\www\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\wamp\www\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\wamp\www\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\wamp\www\yii\framework\web\CController.php(291): CFilterChain->run()
#11 D:\wamp\www\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\wamp\www\yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 D:\wamp\www\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#14 D:\wamp\www\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\wamp\www\yii\demos\blog\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/yii/demos/blog/index.php/post/index
HTTP_REFERER=http://127.0.0.1/yii/demos/blog/index.php/site/login
---
2013/07/09 12:56:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "posts".' in D:\wamp\www\blog\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\wamp\www\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('posts')
#1 D:\wamp\www\blog\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\wamp\www\blog\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/blog/index.php/posts/
---
